### PR TITLE
README adjustments

### PR DESCRIPTION
**Pull request by jamesbraza**
Clearing things up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,18 +23,22 @@ pqa ask 'How can carbon nanotubes be manufactured at a large scale?'
...
 PaperQA2 is engineered to be the best RAG model for working with scientific papers. Here are some features:
 
-- A simple interface to get good answers with grounded responses that have in-text citations.
-- State-of-the-art implementation including metadata-awareness in document embeddings and LLM-based re-ranking and contextual summarization (RCS).
-- The ability to do agentic RAG which iteratively refines queries and answers.
-- Automatic redundant fetching of paper metadata, including citation and journal quality data from multiple providers.
+- A simple interface to get good answers with grounded responses containing in-text citations.
+- State-of-the-art implementation including document metadata-awareness
+  in embeddings and LLM-based re-ranking and contextual summarization (RCS).
+- Answering process is governed by a language agent which iteratively refines queries and answers.
```
Comment (mskarlin) [inline README.md:35]: let's reword this to go back to "agentic RAG"

**Review by mskarlin — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -20,18 +23,23 @@ pqa ask 'How can carbon nanotubes be manufactured at a large scale?'
...
```
Comment (mskarlin) [inline README.md:35]: "Support for agentic RAG, where a language agent can iteratively refine queries and answers."

Comment (jamesbraza) [reply README.md]: Done
